app/views/spree/order_mailer/_order_summary.html.haml

```diff
@@ -43,6 +43,8 @@
     - checkout_adjustments_for(@order, exclude: [:line_item]).reverse_each do |adjustment|
       %tr
         %td{align: "right", colspan: "3"}
+          - if adjustment.originator_type == "Voucher"
+            = "#{t(:voucher)}:"
           = "#{adjustment.label}:"
```

Comment:
May I suggest to use a translation here? Then we can customise the display for each language. Other possibilities could be:

* Voucher CODE:
* CODE (voucher):
* CODE (discount):

You can pass the voucher code to the translation:
```suggestion
          - if adjustment.originator_type == "Voucher"
            = t(".voucher_label", code: adjustment.label)
```

And in `en.yml` I would add "Voucher %{code}:" as translation, avoiding the weird double colon in the line.